project.clj

```diff
@@ -3,5 +3,6 @@
   :url "https://github.com/jank-lang/clojure-test-suite"
   :license {:name "MPL 2.0"
             :url "https://www.mozilla.org/en-US/MPL/2.0/"}
-  :dependencies [[org.clojure/clojure "1.12.0"]]
+  :dependencies [[org.clojure/clojure "1.12.0"]
+                 [org.clojure/clojurescript "1.12.42"]]
```

Comment:
cider doesn't work on this project without this, @jeaye if you want me to do this in a separate PR that's fine but I think we should have it